mkdocs/docs/HPC/connecting.md

```diff
@@ -759,3 +759,17 @@ created.
 ### Fast file transfer for large datasets
 
 See [the section on `rsync` in chapter 5 of the Linux intro manual](../linux-tutorial/uploading_files/#copying-faster-with-rsync).
+
+## Changing login nodes
+It can be useful to work on the same login node using multiple terminal sessions. However, when you connect to the HPC (High-Performance Computing) system through SSH, you're directed to a random login node, which might not be the one where you already have an active session. To address this, there is a way to manually switch your active login node.
```

Comment:
I would rephrase this, it's mostly about having control over which login node you're on, not so much being able to have multiple sessions on the same login node.

The same applies when starting a shell session via the web portal, so I wouldn't mention SSH.

Also try to use "you are" rather than "you're".